api/provider/twitter.go

```diff
@@ -16,7 +16,7 @@ import (
 
 const (
 	requestURL      = "https://api.twitter.com/oauth/request_token"
-	authorizeURL    = "https://api.twitter.com/oauth/authorize"
+	authorizeURL    = "https://api.twitter.com/oauth/authenticate"
```

Comment:
would be good to rename it `authorizeURL` to `authenticateURL` just to be consistent with the endpoint